docs/source/user_section/installation.rst

```diff
@@ -104,7 +104,7 @@ To run the entire testing suite, run ``pytest`` from the
 
   cd ~/shimming-toolbox/shimming-toolbox
   source $HOME/shimming-toolbox/python/etc/profile.d/conda.sh
-  conda activate st_venv
+  conda activate ~/shimming-toolbox/python/     
```

Comment:
`source` should take care of activating the base environment. That's how I was doing it, anyway. So we should just not need this at all.

```suggestion
```